### PR TITLE
publishes to pypi when a tag is pushed

### PR DESCRIPTION
**Pull request by johannesjh**
automatically publishes to pypi when a tag is pushed into the repository.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -59,3 +59,21 @@ jobs:
...
+        uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8 # v3.1.0
+
+      - name: Setup
+        id: setup
+        uses: ./.github/actions/setup
+        with:
+          install-options: --without lint
+
+      - name: Publish to PyPI
+        if: startsWith(github.ref, 'refs/tags/')
```
Comment (real-yfprojects) [inline .github/workflows/ci.yml]: Duplicate if-clause.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -59,3 +59,21 @@ jobs:
         uses: ./.github/actions/publish-docs
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
+
+  publish:
+    runs-on: ubuntu-latest
+    needs: test
+    if: github.event_name == 'push' && contains(github.ref, 'refs/tags/v')
```
Comment (real-yfprojects) [inline .github/workflows/ci.yml]:
```suggestion
    if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags/v')
```

**Review by real-yfprojects — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yml (hunk 2 above)
Comment (johannesjh) [reply .github/workflows/ci.yml]: thank you, I implemented the suggested change.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (johannesjh) [reply .github/workflows/ci.yml]: thank you, I removed the duplicate if clause.

**Review by real-yfprojects — APPROVED** [review]

**Pull request merged**